nomos-mix/core/src/processor/temporal.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct TemporalProcessorSettings {
+    pub max_delay_seconds: u64,
+}
+
+impl Default for TemporalProcessorSettings {
+    fn default() -> Self {
+        Self {
+            max_delay_seconds: 10,
```

Comment:
In the future a configuration parameter, right?

Reply:
This is already a configuration parameter. This code just defines the default value that can be easily used in some tests. But, yeah, actually, I don't think it's a good practice to define default values. Instead, I can let users configure this value always manually. I will remove these defaults.

Reply:
Removed default values: 7091a5ee2210c45d8674f8a3b76bd005440f4e3f